src/workerd/util/BUILD.bazel

```diff
@@ -36,6 +36,7 @@ wd_cc_library(
     visibility = ["//visibility:public"],
     deps = [
         "@sqlite3//:sqlite3",
+        "//src/workerd/jsg",
```

Comment:
As mentioned elsewhere, I'd really prefer to avoid this dependency. This library is intended to be used in some places that aren't running JavaScript.